netlogo-gui/src/main/ide/CodeCompletionPopup.scala

```diff
@@ -38,6 +38,20 @@ case class CodeCompletionPopup() {
       suggestionDisplaylist.setLayoutOrientation(JList.VERTICAL)
       suggestionDisplaylist.setVisibleRowCount(10)
       suggestionDisplaylist.setCellRenderer(new SuggestionListRenderer(eA))
+      
+      suggestionDisplaylist.addMouseListener(new MouseListener {
+        override def mouseExited(e: MouseEvent): Unit = {}
+
+        override def mouseClicked(e: MouseEvent): Unit = {}
+
+        override def mouseEntered(e: MouseEvent): Unit = {}
+
+        override def mousePressed(e: MouseEvent): Unit = {}
+
+        override def mouseReleased(e: MouseEvent): Unit = {
```

Comment:
In the method that is adding the selected suggestion from the list of suggestions, I am making use of the selected element in the list. Now I was not sure will the new element get selected before `mouseClicked` is called or not, but I was pretty sure that the selected element would have changed till `mouseReleased` is called.